caddyhttp/templates/templates.go

```diff
@@ -110,6 +110,10 @@ func (t Templates) ServeHTTP(w http.ResponseWriter, r *http.Request) (int, error
 		// set the actual content length now that the template was executed
 		w.Header().Set("Content-Length", strconv.Itoa(buf.Len()))
 
+		// delete the headers related to cache
```

Comment:
Note for future work: I think a comment for the next two lines is good, but this comment just restates what is obvious from the code. Better would be to have a comment explaining WHY the next two lines are present.

Reply:
FWIW I think it's at least more useful than just stating the obvious, because it's specific domain knowledge that ETag and Last-Modified are cache-related. Not everyone would know that. But I agree that it could explain why those headers are being deleted.